.github/workflows/release.yml

```
@@ -38,3 +38,4 @@ jobs:
       - uses: actions/attest-build-provenance@v2
         with:
           subject-checksums: ./dist/checksums.txt
+          subject-checksums-type: sha256
```

actionlint (1.7.7)

input "subject-checksums-type" is not defined in action "actions/attest-build-provenance@v2". available inputs are "github-token", "push-to-registry", "show-summary", "subject-digest", "subject-name", "subject-path"

(action)